fr/community/contributor-onboarding.md

```diff
@@ -0,0 +1,132 @@
+
+Selon notre [Code de conduite](http://hood.ie/code-of-conduct.html), notre objectif est de fournir un espace sûr pour tous dans cette communauté et nous accueillons avec joie tout le monde, y compris les développeurs et personnes qui souhaitent devenir des contributeurs hors code.
+
+Ce document est et sera toujours en chantier. Nous essayons constamment d'améliorer notre engagement et notre culture communautaire et sommes toujours heureux d'avoir des retours!
+
+###  Qu'est-ce que les gens contribuent et où?
+
+Hoodie est un projet plutôt grand et nous avons plusieurs parties qui ont besoin d'être créées et / ou maintenues. Pour en savoir plus où nous avons besoin d'aide, jetez un oeil sur [le site de contribution](http://hood.ie/contribute/). Si vous ne trouvez rien que vous ayez envie de faire ou avez des questions additionnelles, merci de [nous contacter](http://hood.ie/contact/).
+
+---------------
+
+### 1. Dire bonjour!
+
+- En tant que premier contact pour un nouveau contributeur, vous êtes responsable de les intégrer à la communauté.
+- si ça ne semble pas pertinent, ou si ça n'est pas possible pour quelque raison que ce soir, par exemple vos propres obligations, manque de temps ou raisons personnelles, merci d'en parler à la communauté le plus tôt possible pour voir qui peut être leur contact et les guider dans les semaines et mois à venir.
+- Merci de garder à l'esprit que, idéalement, deux personnes qui partagent le même domaine, les mêmes intérêts ou compétences peuvent apprendre l'un de l'autre. 
+- Une fois que vous avez trouvé quelqu'un, introduisez-les l'un à l'autre.
+
+Essayer de parler au nouveau contributeur. Nous avons appris que ceci permet de lever de nombreux obstacles à la communication et que ça permet aux choses d'avancer plus vite. De plus, rencontrer les gens est toujours amusant!
+
+La manière la plus facile de faire ça est un rapide <a href="http://talky.io" target="_blank">Talky</a>, un appel Skype ou Google Hangout entre la personne intéressée et au moins un contributeur (à commencer par vous). Si vous êtes dans la même ville, aller déjeuner, prendre un café ou une balade est toujours une option préférable.
+
+### 2. Commencer ensembles
+
+1. Présentez-leur notre [Code de Conduite](http://hood.ie/code-of-conduct/). Ne faites pas que le mentionner mais demandez-leur de le lire et demande-leur s'ils ont des questions et inquiétude. Si c'est le cas, répondez-y.
+2. Cernez leur situation:
+  - quelles compétences ont-il?
+     - sont-ils développeurs
+         - frontend / backend / mobile?
+     - font-ils autre-chose que du code?
+         - quel est leur spécialité?
+  - d'où vient leur intérêt dans Hoodie?
+  - sur quoi souhaitent-ils travailler?
+  - quels compétences personnelles les motives?
+  - y a-t-il quelque-chose qu'ils veulent améliorer et / ou pour laquelle on peut les aider à se former?
```

Comment:
Je trouvais "souhaitent" moins catégorique que "veulent", mais c'est définitivement plus proche de "want" - for english speakers: @almet is right, it's "want", not "wish".